game-core/src/main/java/games/strategy/triplea/delegate/battle/BattleState.java

```diff
@@ -35,6 +40,8 @@ public Side getOpposite() {
 
   Collection<Unit> getBombardingUnits();
 
+  Collection<Unit> getAmphibiousLandAttackers();
```

Comment:
While thinking about parameterizing APIs, should we parameterize this as well? Should this even be an API method come to think of it?

What if the caller were to do the filtering it needed on its own leaving the API to be simply "getAttackers()".

Reply:
I thought about doing that as part of this PR but discovered that the `amphibiousLandAttackers` list was being generated in a manner that didn't lend to allowing the caller to generate.  I need to do more investigation and see why it is being done that way and if I can change it without causing problems.